apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClient.cs

```diff
@@ -37,8 +37,14 @@ namespace Google.Cloud.Storage.V1
     /// All instance methods declared in this class are virtual, with an implementation which simply
     /// throws <see cref="NotImplementedException"/>. All these methods are overridden in <see cref="StorageClientImpl"/>.
     /// </para>
+    /// <para>
+    /// This class implements <see cref="IDisposable"/>; however, the <see cref="Dispose"/> method only requires calling
+    /// if many short-lived instances of <see cref="StorageClient"/> are being created.
+    /// Most code would be expected to create a single <c>StorageClient</c> instance, and use this instance throughout
+    /// the lifetime of the application. In this case, <c>Dispose</c> need not be called.
```

Comment:
Even creating multiple clients isn't a problem in many cases, of course. We'll definitely want to have a broader FAQ entry about this, but we can do that later.